### DISCUSSION TITLE
How to get all members in a group

### DISCUSSION THREAD

**Discussion by Kushal7788** [yagop/node-telegram-bot-api#1115] [Q&A]
Are there any methods available to fetch all members in the group? I want userIds of all the members. I see there is a method to get member count in group but no method to actually get a list of all members which returns an Array of [ChatMember] objects.

**Comment by shreya-oeo** [comment]
well i don't know about this module but you can use telegram http api

https://core.telegram.org/bots/api#getchat
 And you will get a chat object in format
https://core.telegram.org/bots/api#chat
 And you can use property active_members inside it

**Reply by melroy89** [reply]
So for this nodeJS package you can use: `telegramBot.getChat()`

**Comment by Abdujabboraka** [comment]
how to check users added members count in telegram group using telebot python

**Reply by melroy89** [reply]
I have the same question. There is a call. But it always returns 1
